### ISSUE TITLE
Cow Bug

### ISSUE THREAD

**Issue by MonoAdrian23** [Nukkit-coders/MobPlugin#234]
### Info

This is:
- [x] bug report
- [ ] feature request
- [ ] other

### Your issue / suggestion
 Cow haves a completly wrong High Calculation. 
It looks like a Babycow but you can melk that and it drops all items like a normal cow. 

MobPlugin 1.20.4

### Configuration
Link to debugpaste:
```
<!-- Run debugpaste command and put the link here -->
```
Config:
```
<!-- Put your config.yml contents here  -->
```

**Comment by MonoAdrian23** [comment]
![IMG_20210706_200617.jpg](https://user-images.githubusercontent.com/59027482/124647461-48a27880-de96-11eb-8be5-dda58b01378b.jpg)

**Comment by PetteriM1** [comment]
Should be fixed in 1.20.6
